host_applications/linux/apps/raspicam/RaspiStill.c

```diff
@@ -856,7 +709,7 @@ static int parse_cmdline(int argc, const char **argv, RASPISTILL_STATE *state)
  *
  * @param app_name String to display as the application name
  */
-static void display_valid_parameters(char *app_name)
+void application_help_message(char *app_name)
```

Comment:
If passed as a function pointer to the helper function then this can remain static.